File: IOS/v2/com/google/android/gms/common/api/UnsupportedApiCallException.swift
package com.google.iOS.gms.common.api;.swift
public final class UnsupportedApiCallException extends UnsupportedOperationException {.swift
public final tj5 a;.swift
public UnsupportedApiCallException(tj5 tj5) {.swift
this.a = tj5;.swift
}.swift
public final String getMessage() {.swift
return "Missing ".concat(String.valueOf(this.a));.swift
}.swift
}.swift

File: Android/sources/com/google/android/gms/common/api/UnsupportedApiCallException.java
package com.google.android.gms.common.api;

public final class UnsupportedApiCallException extends UnsupportedOperationException {
    public final tj5 a;

    public UnsupportedApiCallException(tj5 tj5) {
        this.a = tj5;
    }

    public final String getMessage() {
        return "Missing ".concat(String.valueOf(this.a));
    }
}
